retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss

```diff
@@ -2335,6 +2335,14 @@ QFrame#titleBarFrame QTextEdit {
 
 /**** Special Page tweak ****/
 
+/* Channels */
+
+CreateGxsChannelMsg QPushButton#channelpostButton, QPushButton#attachmentsButton,
+QPushButton#addThumbnailButton {
```

Comment:
Prefer use generic names like btnWI_* (button With Image) so no need to add them one by one and get a big (messy) qss file.

Reply:
oh you want that i change the names of the buttons? but i want only fix the issues without touch ui files.
normaly when some one touch to fix the qss styles he need touch edit the code too?

Reply:
I don't want qss files becomes as messy as they were.
Here, you don't want to fix one thing but general one. So fix it in general way.

Reply:
how to use generic names in qss? i dont understand that any example in qss how need to be look

like this:
```
QFrame[objectName^="feedFrame"],/* Frame used in Feeds*/
QFrame[objectName^="plainFrame"],/* Frame used in Widget with plain background*/
QFrame[objectName^="plainBFrame"] {/* Frame used in Widget with plain bordered background*/
	background-color: #19232D;
}
```

Reply:
but you doesnt fixed the buttons look and feel issues  in light & dark style

Reply:
like this?:
QPushButton[objectName^="channelpostButton"] {
}

Reply:
Rename buttons with image like "btnWI_ChannelPost"
In qss use:
QPushButton[objectName^="btnWI_"] {
...
}

Reply:
ok mean renaname first

Reply:
im wondering why there are lots of issues on the buttons with spacing

![image](https://user-images.githubusercontent.com/9952056/145682427-01d8b4a0-b295-4a53-8cab-f9c2194bb86f.png)
![image](https://user-images.githubusercontent.com/9952056/145682441-007b6886-7246-49e8-9f2b-0698b318cccc.png)
![image](https://user-images.githubusercontent.com/9952056/145682460-cf4aba1b-b46e-4460-8102-7e09c17f0821.png)
![image](https://user-images.githubusercontent.com/9952056/145682523-f7240eb2-6a47-40d8-bfa9-0db4f3457db6.png)

Reply:
"Cancel" button looks not normal too

Reply:
seems issues more with qpushbuttons, on qtoolbuttons they have spacing with pictures

Reply:
The fact that cancel button are smaller don't hurt me, on the contrary.